Doc/using/cmdline.rst

```diff
@@ -302,16 +302,22 @@ Miscellaneous options
 
 .. option:: -i
 
-   Enter interactive mode after execution.
+   Enter interactive mode after execution, or force interactive mode even when
+   :data:`sys.stdin` does not appear to be a terminal.
```

Comment:
I would probably rewrite it as it is worded in `python --help`.

> forces a prompt even if stdin does not appear to be a terminal

Saying "interactive" is not quite correct, "prompt" better describes it.